aten/src/ATen/gen.py

```diff
@@ -180,7 +181,7 @@ def backend_to_devicetype(backend):
         return 'CPU'
     return backend
 
-backends = ['CPU', 'CUDA', 'QuantizedCPU']
+backends = ['CPU', 'CUDA']
```

Comment:
can you clean these up in the future?  These aren't backends, they are device types.